src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -99,5 +99,13 @@ public override bool IsExplicitMatch(ITest test)
 
         //    return false;
         //}	
+
+        public override TNode AddToXml(TNode parentNode, bool recursive)
+        {
+            TNode result = parentNode.AddElement("not");
+            if (recursive)
+                BaseFilter.AddToXml(result, true);
+            return result;
+        }
     }
```

Comment:
@oznetmaster, I think I made a mistake, I see your comment in the overall listing now, but it is added to the old side of the diff. I guess I missed it, my apologies...